packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.test.ts

```diff
@@ -117,4 +126,109 @@ describe('platform_browser/recaptcha/recaptcha_enterprise_verifier', () => {
       expect(await verifier.verify()).to.eq(FAKE_TOKEN);
     });
   });
+
+  context('handleRecaptchaFlow', () => {
+    let mockAuthInstance: AuthInternal;
+    let mockRequest: any;
+    let mockActionMethod: sinon.SinonStub;
+
+    beforeEach(async () => {
+      mockAuthInstance = await testAuth();
+      mockRequest = {};
+      mockActionMethod = sinon.stub();
+    });
+
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('should call actionMethod with request if emailPasswordEnabled is true', async () => {
+      if (typeof window === 'undefined') {
+        return;
+      }
+      sinon
+        .stub(mockAuthInstance, '_getRecaptchaConfig')
+        .returns({ emailPasswordEnabled: true, siteKey: 'mock_site_key' });
+      sinon
+        .stub(RecaptchaEnterpriseVerifier.prototype, 'verify')
+        .resolves('recaptcha-response');
+      mockRequest = { foo: 'bar' };
+      mockActionMethod = sinon.stub().resolves('testResponse');
+      const response = await handleRecaptchaFlow(
+        mockAuthInstance,
+        mockRequest,
+        RecaptchaActionName.SIGN_IN_WITH_PASSWORD,
+        mockActionMethod
+      );
+      expect(mockActionMethod).to.have.been.calledOnce;
+      expect(response).to.equal('testResponse');
+    });
+
+    it('should handle MISSING_RECAPTCHA_TOKEN error when emailPasswordEnabled is false', async () => {
```

Comment:
This error will be handled even if "emailPasswordEnabled" is true or if the config does not exist too, right?

Maybe add a comment? 

"Errors like "MISSING_RECAPTCHA_TOKEN" will be handled irrespective of the enablement status of "emailPasswordEnabled", but this test verifies the more likely scenario where emailPasswordEnabled is false"

Reply:
Yeah, it's not necessarily "emailPasswordEnabled" is false. Added the comment.